From c821f2e7f2534ece24a10402df3f501536a09cbd Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Wed, 28 Jan 2009 15:00:35 +0100
Subject: [PATCH] Fix off_t in kernel struct

The off_t in kernel struct (for ioctls) is actually different from the
definition of user-space off_t.  The kernel off_t is equial with long
while user-space off_t depends on the large-file support.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 include/sound/asound.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/sound/asound.h b/include/sound/asound.h
index 977b2d6..62d1e57 100644
--- a/include/sound/asound.h
+++ b/include/sound/asound.h
@@ -402,7 +402,7 @@ struct sndrv_pcm_sw_params {
 
 struct sndrv_pcm_channel_info {
 	unsigned int channel;
-	off_t offset;			/* mmap offset */
+	long offset;			/* mmap offset */
 	unsigned int first;		/* offset to first sample in bits */
 	unsigned int step;		/* samples distance in bits */
 };
-- 
1.5.6.5

